### PR TITLE
Update SortableElementProps definition to contain the value field

### PR DESCRIPTION
**Pull request by andrewdewaal**
Please fill in this template.

- [ ] Make your PR against the `master` branch.
- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/clauderic/react-sortable-hoc/issues/117>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

**Comment by sheetalkamat** [comment]
@andrewdewaal Please add a test for your scenario

File: react-sortable-hoc/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for nes 0.0.7
 // Project: https://github.com/clauderic/react-sortable-hoc
-// Definitions by: Ivo Stratev <https://github.com/NoHomey>
+// Definitions by: Andrew de Waal <https://github.com/andrewdewaal>
```
Comment (NoHomey) [inline react-sortable-hoc/index.d.ts]: Definitions by should be list of contributors contributed to the given Definition, please re-add my information as contributor! And please in the future do not remove contributor information just by changing one line ...

Comment (NoHomey) [inline react-sortable-hoc/index.d.ts]: Here is an example how it should [look](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3153e0fb9982cd5135d141a0b6d8aa84e33be4e8/jest/index.d.ts#L3)

**Comment by andrewdewaal** [comment]
Apologies for the delay; I was actually able to find another solution that does not require a change to the definition. Turns out I was using the API incorrectly :)

**Comment by mhauens** [comment]
@andrewdewaal Can you please provide me the solution you found? I'm running into the same Problem. If I add value?: any to SortableContainerProps and to SortableElementProps it would work fine.

**Comment by mhauens** [comment]
Figured it out!


    import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc';
    
    const SortableItem = SortableElement((item: any) =>
          
    );
     
    const SortableList = SortableContainer((adImageList: any) =>
            <section>
              {adImageList["items"].map((value: any, index: any) =>
                <SortableItem key={"item-${index}"} index={index} value={value} />
              )}
            </section>
    );

    _onSortEnd = (SortEnd: any) => {
        this.setState({
            adImageList: arrayMove(this.state.adImageList, SortEnd.oldIndex, SortEnd.newIndex),
        });
    };

    <SortableList onSortEnd={this._onSortEnd} axis="x" items={this.state.adImageList} lockToContainerEdges={true}/>

**Pull request closed without merging**